File: Examples/CSharp/ModifyingAndConvertingImages/JPEG/RotatingAnImage.cs
/*
This project uses the Automatic Package Restore feature of NuGet to resolve the Aspose.Imaging for .NET API reference 
when the project is built. Please check https://learn.microsoft.com/en-us/nuget/resources/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Imaging for .NET API from https://releases.aspose.com/, 
install it, and then add its reference to this project. For any issues, questions, or suggestions, 
please feel free to contact us using https://forum.aspose.com/
*/

using System;
namespace Aspose.Imaging.Examples.CSharp.ModifyingAndConvertingImages.JPEG
{
    class RotatingAnImage
    {
        public static void Run()
        {
            //ExStart:RotatingAnImage
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_JPEG();

            Console.WriteLine("Running example RotatingAnImage");
            // Loading and rotating the image
            using (var image = Image.Load(dataDir + "aspose-logo.jpg"))
            {
                image.RotateFlip(RotateFlipType.Rotate270FlipNone);
                image.Save(dataDir + "RotatingAnImage_out.jpg");
            }

            Console.WriteLine("Finished example RotatingAnImage");
            //ExEnd:RotatingAnImage
        }       
    }
}

File: Examples/VisualBasic/ModifyingAndConvertingImages/JPEG/RotatingAnImage.vb
Imports Aspose.Imaging.ImageOptions

'
'This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Imaging for .NET API reference 
'when the project is build. Please check https://Docs.nuget.org/consume/nuget-faq for more information. 
'If you do not wish to use NuGet, you can manually download Aspose.Imaging for .NET API from http://www.aspose.com/downloads, 
'Install it and then add its reference to this project. For any issues, questions or suggestions 
'Please feel free to contact us using http://www.aspose.com/community/forums/default.aspx
'

Namespace Aspose.Imaging.Examples.VisualBasic.ModifyingAndConvertingImages.JPEG
    Class RotatingAnImage
        Public Shared Sub Run()
            ' ExStart:RotatingAnImage
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_JPEG()

            ' Loading and Rotating an Image
            Using image__1 = Image.Load(dataDir & Convert.ToString("aspose-logo.jpg"))
                image__1.RotateFlip(RotateFlipType.Rotate270FlipNone)
                image__1.Save(dataDir & Convert.ToString("RotatingAnImage_out.jpg"))
            End Using
            ' ExEnd:RotatingAnImage
        End Sub
    End Class
End Namespace